src/features/Travel/Main.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+
+import { Hacker, Travel } from '../../api';
+import { HACKATHON_NAME, IHacker, ITravel } from '../../config';
+import { BackgroundImage, H1, H2, MaxWidthBox, Button } from '../../shared/Elements';
+
+import ValidationErrorGenerator from '../../shared/Form/validationErrorGenerator';
+import WithToasterContainer from '../../shared/HOC/withToaster';
+
+import Train from '../../assets/images/train.svg';
+
+export interface ITravelState {
+  hacker: IHacker | null;
+  travel: ITravel | null;
+  isLoading: boolean;
+}
+
+/**
+ * Container that renders form to log in.
+ */
+class TravelContainer extends React.Component<{}, ITravelState> {
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      hacker: null,
+      travel: null,
+      isLoading: true
+    };
+  }
+  public render() {
+    let reimbursement = <div></div>;
+    if (this.state.travel) {
+      switch (this.state.travel.status) {
+        case 'None':
+          reimbursement = <div>
+            Your request to recieve ${this.state.travel.request.toFixed(2)} in reimbursement for travel is still being processed.
+          </div>;
+          break;
+        case 'Bus':
+          reimbursement = <div>
+            You are taking a bus. We should put more info about the bus here.
+          </div>;
+          break;
+        case 'Policy':
+          reimbursement = <div>
+            Your travel reimbursement decision has been released. In order to see how much you will be reimbursed, you must first
+            agree to our <a href="https://docs.google.com/document/d/1K8WSGQtWfKrybT_O9WrxIp93dETrv3jhy71fkHKZwdM" target="_blank">travel policy</a>.
+            <div style={{ textAlign: 'center', marginTop: '18px' }}>
+              <Button>I agree to McHacks Travel Policy</Button>
+            </div>
+          </div>;
+          break;
+        case 'Offered':
+        case 'Valid':
+        case 'Invalid':
+          // TODO: Handle Valid and Invalid cases once reciepts are handled
+          if (this.state.travel.offer > 0) {
+            reimbursement = <div>
+              We will reimburse you for
```

Comment:
I'll update all the text to constants file later with deadlines and such. Looks good :)